packages/sdk/src/types/Vouchers.ts

```diff
@@ -2,7 +2,7 @@ import { OrdersGetResponse } from './Orders'
 import { SimpleCustomer } from './Customers'
 import { DiscountUnit, DiscountAmount, DiscountPercent } from './DiscountVoucher'
 
-export type VoucherType = 'GIFT' | 'DISCOUNT' | 'LOYALTY_CARD' | 'LUCKY_DRAW'
+export type VoucherType = 'GIFT' | 'DISCOUNT_VOUCHER' | 'LOYALTY_CARD' | 'LUCKY_DRAW'
```

Comment:
Dear @Irene350 could you also change `GIFT` to `GIFT_VOUCHER`? Unfortunately we overlook this when doing first iteration for this SDK.

Reply:
sure thanks for double checking!